gravitee-apim-repository/gravitee-apim-repository-test/src/test/resources/data/portalnavigationitem-tests/portalNavigationItems.json

```diff
@@ -20,6 +20,16 @@
     "parentId": null,
     "order": 2,
     "configuration": "{ \"url\": \"https://docs.example.com\" }"
+  },
+  {
+    "id": "nav-item-3",
+    "organizationId": "org-1",
+    "environmentId": "env-1",
+    "title": "Sub Home",
+    "type": "PAGE",
+    "area": "HOMEPAGE",
+    "parentId": "nav-item-1",
+    "order": 1,
+    "configuration": "{ \"pageId\": \"sub-home\" }"
```

Comment:
Even if it's ok for testing, the data are not really relevant.
Could you try to add a Folder and put this page in the folder?